src/core_plugins/console/public/index.html

```diff
@@ -27,10 +28,10 @@
             aria-labelledby="consoleRequestOptions"
```

Comment:
I assume, you doesn't mean the `ul`, but got confused with the lines (most likely because GitHub collapsed the code in the middle of the element :D) I anyway made both `a.editor-action` also into `button`s.